cfme/tests/intelligence/test_optimization.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from widgetastic.utils import attributize_string
+
+from cfme import test_requirements
+from cfme.utils.blockers import BZ
+
+pytestmark = [test_requirements.report, pytest.mark.tier(1), pytest.mark.ignore_stream("5.10")]
+
+REPORTS = [
```

Comment:
Optional for this PR, but consider defining this list with the report entity classes, if they're consistent report types that may (will) be used in multiple tests. If they're just specific parametrization report types for this test module, then ignore me :)

Reply:
Ignoring you. :grimacing: